lib/Models/TableCatalogItem.js

```diff
@@ -448,10 +448,10 @@ TableCatalogItem.prototype.initializeFromTableStructure = function(tableStructur
                 if (addressGeocoderData.missingAddresses.length > 0) {
                     missingAddressesMessage = "\nThe CSV contains addresses, but " + addressGeocoderData.missingAddresses.length
                         + " can\'t be located on the map:\n"
-                        + addressGeocoderData.missingAddresses.join(", ") + "\n";
+                        + addressGeocoderData.missingAddresses.join(", ") + ".</br></br>";
```

Comment:
Should be `<br />`?  Two `\n`s should do the trick too because of the Markdown formatting.